apps/web/app/.server/service/l2-cancellations.ts

```diff
@@ -52,18 +59,23 @@ function blocksInADay() {
 }
 
 async function getL2ProposalState(address: Hex, proposalId: Hex): Promise<L2_CANCELLATION_STATES> {
-  return l2Rpc.contractRead(address, "state", ZK_GOV_OPS_GOVERNOR_ABI, z.number(), [proposalId]);
+  return l2Rpc.contractRead(address, "state", zkGovOpsGovernorAbi, z.number(), [proposalId]);
 }
 
 async function fetchProposalsFromL2Governor(type: L2CancellationType, from: bigint) {
   const proposalCreatedLogsPromise = l2Rpc
     .getLogs(getL2GovernorAddress(type), numberToHex(from), "latest", [
-      zkGovOpsGovernorAbi.eventIdFor("ProposalCreated"),
+      toEventSelector(
+        getAbiItem({
+          abi: zkGovOpsGovernorAbi,
+          name: "ProposalCreated",
+        })
+      ),
```

Comment:
Maybe this can be extracted to a function.

Reply:
It's only used in three times in the codebase, we can skip it for now and come back later if we need it.